src/server.js

```diff
@@ -104,42 +104,77 @@ app.get('/list', (req, res) => {
 app.get('/orgform', (req, res) => {
   res.render('orgform');
 });
-
+// addrole- its deal with orgform and we validate orgform
 app.post('/addrole', (req, res) => {
-  req.checkBody('org_name', 'Organisation name required').notEmpty().isAlpha();
-  const errors = req.validationErrors();
-  if (errors) {
-    console.log(errors);
-    res.redirect('/orgform');
-  } else {
-    MongoClient.connect(url, (err, db) => {
-      if (err) return ('Error connection to DB: ', err);
-      else {
-        console.log('connection made');
-        // object take the data from html page and put in this object
-        const role = {
-          'org_name': req.body.org_name,
-          'org_desc': req.body.org_desc,
-          'phone_num': req.body.user_phone,
-          'email': req.body.user_mail,
-          'role_name': req.body.role_name,
-          'role_desc': req.body.role_desc,
-          'num_vlntr_req': req.body.num_vol,
-          'start_date': req.body.start_date,
-          'end_date': req.body.end_date
-        };
-        // connect to the table called vol_roles
-        const collection = db.collection('vol_roles');
-        // insert the data in db
-        collection.insert(role, {w: 1}, (err, result) => {
-          if (err) return ('Error inserting to DB: ', err);
-          db.close();
-          // redirect the information to the list page also
-          res.redirect('/list');
-        });
-      }
-    });
-  }
+  req.checkBody('org_name', 'Organisation name required').notEmpty();
+  // -------------------------------------------
+  req.checkBody('org_desc', 'Organisation description required').notEmpty();
+  // ---------------------------------------
+  req.checkBody('user_phone', 'Phone number required').notEmpty();
+  req.checkBody('user_phone', 'Phone number not valid (must only contain numbers').isInt();
+  req.checkBody('user_phone', 'Phone number not valid (must only contain 10 digits').isLength({min: 10, max: 10});
+  req.checkBody('user_phone', 'Phone number not valid').isNumeric();
+  // ---------------------------------------------
+  req.checkBody('user_mail', 'Email required').notEmpty();
```

Comment:
I don't know if all organisations will have an email address do you think we should make this a required field?